recipe/meta.yaml

```diff
@@ -1,43 +1,62 @@
 {% set name = "iopath" %}
-{% set version = "0.1.9" %}
+{% set version = "0.1.10" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/facebookresearch/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 3f17a6558fa62fdc2ab1ecdf7c106576167fb8bb19f1612289fd88467535ad7a
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/iopath-{{ version }}.tar.gz
+  sha256: 3311c16a4d9137223e20f141655759933e1eda24f8bff166af834af3c645ef01
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: true  # [py<=36]
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
+    - setuptools
+    - wheel
   run:
-    - dataclasses
-    - portalocker
-    - python >=3.6
+    - python
     - tqdm
+    - typing_extensions
+    - portalocker
+    - dataclasses  # [py<37]
+
+{% set ignore_tests = "" %}
+# $PREFIX/lib/python3.10/site-packages/_pytest/threadexception.py:73: PytestUnhandledThreadExceptionWarning: Exception in thread Thread-1 (_poll_jobs)
+{% set ignore_tests = " --ignore=tests/async_torch_test.py" %}
```

Comment:
`async_torch_test.py` requires `pytorch`, did you try with that? Which test is giving you `PytestUnhandledThreadExceptionWarning`? Do you have a log, or a commit id where the error was happening on Prefect?